data/NV/authorities.json

```diff
@@ -47,84 +47,84 @@
   "county": {
     "nv-washoe-county": {
       "name": "Washoe County",
-      "county": "Washoe"
+      "county_fips": "32031"
```

Comment:
Does this fips need to be set by developers or is the `generate-misc-state-data.ts` script taking care of this? 

Is any documentation necessary to inform developers on how to pull this value if it isn't automated?

Reply:
It needs to be set by engineers. (Setting the `county` key was also done manually.) I'll add something to the readme